app/views/posts/show.html.erb

```diff
@@ -14,6 +13,7 @@
         <%= link_to 'Edit', edit_post_path(@post), class: "btn btn-sm btn-warning" %>
         <%= link_to 'Back' , user_path(user), class: "btn btn-sm btn-warning" %>
       </div>
+      </div>
```

Comment:
is this extraneous? I don't see a new opening </div> in this commit 🤔

Reply:
Oof it is, I'll fix it.